src/sqlancer/feldera/FelderaSchema.java

```diff
@@ -129,29 +114,144 @@ public FelderaExpression getRandomConstant(FelderaGlobalState globalState) {
 
             return FelderaConstant.getRandomConstant(globalState, this);
         }
+
+        public boolean isNumeric() {
```

Comment:
This looks wrong.
An array is not numeric.

Reply:
This method is not specific to arrays.
It returns true for int types, FP types and decimal.